docs/_interface/sync-modes.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Sync-modes
+sort-key: L
+---
+
+Syncing is the process by which Geth catches up to the latest Ethereum block and current global state. 
+There are several ways to sync a Geth node that differ in their speed, storage requirements and trust 
+assumptions. This page outlines three sync configurations for full nodes and one for light nodes.
+
+## Full nodes
+
+There are two types of full node that use different mechanisms to sync up to the head of the chain:
+
+### Snap (default)
+
+A snap sync'd node holds the most recent 128 blocks in memory, so transactions in that range are always 
```

Comment:
the most recent 128 block states in memory. And transactions are always accessible I think :)